open-apparel-registry/open-apparel-registry
# Fix contributor types

## Overview

When there is a contributor type in CONTRIB_TYPE_CHOICES which does not
any existing contributors of that type, the code is throwing a
DoesNotExist error and returning a 500 to the UI.

This repairs that issue by catching the DoesNotExist error,
and instead returning a count of 0 for that contributor type, which
is the expected behavior.

## Testing Instructions

(Demonstrate error on develop locally:)
* On develop, add a nonexistent contributor type to CONTRIB_TYPE_CHOICES in models.py
* Run `vagrant ssh` and `./scripts/server`
* When you open the contrib types list, you should get a 500 error
(Test error fix:)
* Checkout this branch
* Add a nonexistent contributor type to CONTRIB_TYPE_CHOICES
* Run `vagrant ssh` and `./scripts/server`
* When you open the contrib types list, you should see a [0] count for your nonexistent contrib type, and you should not have an error

## Checklist
